packages/database/lib/migrations/20250829190900_create_connect-ui-settings_table.cjs

```diff
@@ -0,0 +1,21 @@
+exports.config = { transaction: true };
+
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function (knex) {
+    await knex.schema.createTable('connect_ui_settings', (table) => {
+        table.increments('id').primary();
+        table.integer('environment_id').references('id').inTable('_nango_environments').onDelete('CASCADE').notNullable();
+        table.jsonb('theme').notNullable();
```

Comment:
are you planning on filtering based on this column? otherwise just `json` is enough tbh